### DISCUSSION TITLE
增加一个非酋黑主题

### DISCUSSION THREAD

**Discussion by whitecat346** [Meloong-Git/PCL#4581] [讨论]
有了欧皇彩怎么能少了非酋黑呢！_（其实只是为了给非酋的自已一个慰藉）_

可能会跟玄素黑出现冲突，设计会比较难，我觉得可以偏向神色橡木的颜色，这样区分度比较高（虽然看起来就像棕色了……）

欢迎各位积极讨论！╰(*°▽°*)╯

**Comment by Deep-Dark-Forest** [comment]
>  | 有关拓展隐藏主题、自定义主题、背景图片功能的 Issue | 备注：此类 Issue 大概率会因为必要性及优先级较低被拒绝，但不排除个别会被采纳，仅作参考。<br> *搜索关键词：毛玻璃、背景、透明、标题、隐藏主题、音乐、图标、暗黑、RGB、渐变、图标。* | #3202 #2216 #1487 #1653 #1242 #840 #791 #711 #562 #401 #131

#1930

大概率被拒，或许可以让龙猫看看……？
